festim/generic_simulation.py

```diff
@@ -188,7 +188,19 @@ def attribute_source_terms(self):
 
         # set sources
         for source in self.sources:
-            field_to_object[source.field].sources.append(source)
+            if isinstance(source, festim.RadioactiveDecay) and source.field == "all":
+                # this could be refactored if sources accept several fields
+                # (e.g. festim.Source(field=["0", "T"]))
+                # for field in source.field:
+                #    field_to_object[field].sources.append(source)
```

Comment:
Think this could be removed an put into an issue no?

Reply:
I was wondering if I would do it in this PR or not

Reply:
Maybe it's not worth it, i'll just remove the comment

Reply:
It's not worth it actually, I'll simply remove it